os/net/net_initialize_netmgr.c

```diff
@@ -99,6 +102,12 @@ void net_setup(void)
 
 void net_initialize(void)
 {
+#ifdef CONFIG_NET_LOCAL
+	/* Initialize the local, "Unix domain" socket support */
+
+	local_initialize();
```

Comment:
is this function defined utils.h?

Reply:
This function is defined at local/local.h